docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-bootstrap-docs",
-  "version": "0.0.0",
+  "version": "6.1.0",
```

Comment:
Why do we need to bump the version? When do we need to bump it in future?

Reply:
I actually think the version is not needed at all here. I've removed it since it is a private package anyway